statistic-labels/index.html

```diff
@@ -338,6 +339,11 @@ <h3 id="printBasinTypeHeader">Basin Characteristic Types</h3>
 			  <select onChange="filterByStatGroup(this.value)" id="statTypeDropdown">
 				<option value="None">None</option>
 			  </select>
+			  <br>
+			  <label>Filter by Study Area:</label>
```

Comment:
Consider changing to "Filter by Region"? Unless there is a reason we are using the "Study Area" wording. "Region" is more consistent with the StreamStats client.